src/mjbots_robot_interface.cpp

```diff
@@ -181,7 +184,13 @@ void MjbotsRobotInterface::SetModeStop() {
 void MjbotsRobotInterface::Shutdown() {
   moteus_interface_->shutdown();
 }
-::mjbots::pi3hat::Attitude MjbotsRobotInterface::GetAttitude() {
-  return  attitude_;
+
+const ::kodlab::Attitude<float>& MjbotsRobotInterface::GetAttitude() {
+  return *attitude_;
```

Comment:
Could be in the header, but fine here too

Reply:
Good point, probably worth moving.

Reply:
On second thought, several other accessors are in the implementation as well.  This move should be addressed in a seperate PR.